storage/tokudb/tokudb_information_schema.cc

```diff
@@ -1119,9 +1119,9 @@ void background_job_status_callback(
     table->field[3]->store(type, strlen(type), system_charset_info);
     table->field[4]->store(params, strlen(params), system_charset_info);
     if (user_scheduled)
-        table->field[5]->store("USER", sizeof("USER"), system_charset_info);
+        table->field[5]->store("USER", strlen("USER"), system_charset_info);
```

Comment:
Should this go to 5.6 as well? Not right now with an upmerge or anything like that, just to make a note somewhere right now